src/containers/Settings/UserSettings.js

```diff
@@ -267,7 +267,11 @@ const mapStateToProps = state => {
   };
 };
 
-UserSettingsComponent = reduxForm({ form: 'settings' })(UserSettingsComponent);
+UserSettingsComponent = reduxForm({
+  form: 'settings',
+  enableReinitialize: true,
+  destroyOnUnmount: false
+})(UserSettingsComponent);
 
```

Comment:
see any issues with these redux form settings? this seemed to fix the issue where the fields would get cleared out